ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/SandboxServer.scala

```diff
@@ -138,6 +138,7 @@ final class SandboxServer(
       Files.createDirectories(profileDir)
       engine.startProfiling(profileDir)
   }
+  engine.enableStackTraces(config.stackTraces)
```

Comment:
@SamirTalwar-DA Is that enough to make it work in `sandbox-next`? Or does it only affect `sandbox-classic`?

Reply:
Oh, good catch. This will only affect Sandbox Classic. You'll need to add it to `com.daml.platform.sandboxnext.Runner` too.

Reply:
Thanks for letting me know. I'll do that in a followup PR together with the profiler as well.